container/container.go

```diff
@@ -436,17 +436,25 @@ func (w *ContainerWorker) finalizeDeployment(agreementId string, deployment *con
 		// the cgoup permission can be omitted. It defaults to "rwm" when omitted.
 		for _, givenDevice := range service.Devices {
 			cgp := "rwm"
+			pic := ""
 			sp := strings.Split(givenDevice, ":")
 			if len(sp) == 3 {
 				// the cgroup permission
 				cgp = sp[2]
-			} else if len(sp) < 2 || len(sp) > 3 {
+				pic = sp[1]
+			} else if len(sp) == 2 {
+				cgp = cgp
+				pic = sp[1]
+			}else if len(sp) < 1{
```

Comment:
should be `len(sp) <= 0`

Reply:
Ohh. Added by mistake!

Reply:
Sorry, my mistake in the comment. The above line should be `else if len(sp) == 1`

Reply:
@linggao yes that was I wondering about if we will put len(sp)<1 that implies sp is empty then how we can use docker to generate the container it will simply throw an error